repository/population.go

```diff
@@ -0,0 +1,126 @@
+package repository
+
+import (
+	"fmt"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/sirupsen/logrus"
+)
+
+type populationRepository struct {
+	mysql *sqlx.DB
+}
+
+type PopulationRepository interface {
+	QueryAllDistrict() ([]ResultQueryDistrictIdAndName, error)
+	QueryTotalPopulation(districtId string) (ResultQueryTotal, error)
+	QueryPeopleCommitedTheVote(districtId string) (ResultQueryPeopleCommitTheVote, error)
+	QueryPeopleRightToVote(districtId string) (ResultQueryPeopleWithRightToVote, error)
+}
+
+func NewPopulationRepository(mysql *sqlx.DB) PopulationRepository {
+	return &populationRepository{
+		mysql: mysql,
+	}
+}
+
+type ResultQueryDistrictIdAndName struct {
+	LocationID string `db:"DistrictID"`
+	Location   string `db:"Name"`
+}
+
+type ResultQueryTotal struct {
+	Total int64 `db:"Total"`
+}
+type ResultQueryPeopleWithRightToVote struct {
+	PeopleWithRightToVote int64 `db:"HaveRight"`
+}
+type ResultQueryPeopleCommitTheVote struct {
+	PeopleCommitTheVote int64 `db:"Commits"`
+}
```

Comment:
This struct seems to be too specific. Please see the comment in function where you use it below krub.